From: Marek Vasut <marex@denx.de>
Date: Wed, 13 Jan 2016 18:12:58 +0100
Subject: Add new lock-obj-pub for NIOS2.

src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h: New.
---
 src/Makefile.am                                   |  1 +
 src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h | 23 +++++++++++++++++++++++
 2 files changed, 24 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h

diff --git a/src/Makefile.am b/src/Makefile.am
index e4fe305..8aca7df 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -56,6 +56,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.mips-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h \
         syscfg/lock-obj-pub.mipsel-unknown-linux-gnu.h      \
+	syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h       \
         syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.powerpc64-unknown-linux-gnu.h   \
diff --git a/src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h
new file mode 100644
index 0000000..dbd46da
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.nios2-unknown-linux-gnu.h
@@ -0,0 +1,23 @@
+## lock-obj-pub.nios2-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[24];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
